docs/tutorial/setting-up/index.md

```diff
@@ -40,7 +40,8 @@ A *codespace* is a development environment that's hosted in the cloud.
 
 1. Click **Create codespace on main**
 
-The codespace will open in a new tab.
+It will take roughly two minutes to create the codespace,
+which will open in a new tab.
```

Comment:
Is this a one-time cost?

Reply:
Yes, in the once-per-codespace sense. In other words, if you do the tutorial twice, in different codespaces, then you pay twice; if you do the tutorial twice, in the same codespace, then you pay once.

I investigated [prebuilding][1], but

> If it currently takes more than 2 minutes to create a codespace for a repository, you are likely to benefit from using prebuilds.

So, it seems we are unlikely to benefit from using prebuilds. Even if we were, then I'm not sure that we could, because the prebuild would be associated with the template repo, not the repo the user created from the template repo.

[1]: https://docs.github.com/en/codespaces/prebuilding-your-codespaces